documentation/src/main/asciidoc/core/manual/en_US/18_jpql_functions.adoc

```diff
@@ -300,6 +300,38 @@ Aggregates/concatenates the values produced by `expression` to a single string s
 
 WARNING: This function might not be supported by all DBMS, so make sure your target database does before using it
 
+===== WINDOW functions
```

Comment:
Give a warning that this is an internal function and the syntax might change at any time. Refer to chapter 21 where the supported syntax is explained. See the PAGE_POSITION function description for a warning example